types/underscore/underscore-tests.ts

```diff
@@ -2978,6 +2978,17 @@ _.defer(() => alert("deferred")); // $ExpectType void
     welcome("moe"); // $ExpectType any
 }
 
+// restArguments
+
+// Returns a version of the function that, when called, receives all arguments from and beyond startIndex collected into a single array.
+{
+    const raceResults = _.restArguments((words: string[]) => words.join(''));
+    raceResults("Dopey", "Grumpy", "Happy", "Sneezy", "Bashful", "Sleepy", "Doc");
+    const startIndex = 2;
+    const raceHasStartIndexResults = _.restArguments((words: string[], startIndex) => words.join(''));
```

Comment:
This is not how `startIndex` is used. Please have another look at the documentation and feel free to ask for clarification.

Reply:
> This is not how is used. Please have another look at the documentation and feel free to ask for clarification.`startIndex`
Is that how you use it?
const startIndex = 2;
_.restArguments((words: string[]) => words.join (''), startIndex)

Reply:
> This is not how is used. Please have another look at the documentation and feel free to ask for clarification.`startIndex`

Is that how you use it?
const startIndex = 2;
_.restArguments((words: string[]) => words.join (''), startIndex)